flatpak-builder/action.yml

```diff
@@ -8,6 +8,11 @@ inputs:
   manifest-path:
     description: "The relative path of the manifest file."
     required: true
+  stop-at-module:
+    description: >
+      Stop at the specified module, ignoring it and all the following ones.
+      Note that using this option enforces `build-bundle` to false.
```

Comment:
Nitpick: please remove `Note that` from the sentence

Reply:
Done